resources/migrations/002-friend-requests.up.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE users (
+    id                      uuid DEFAULT gen_random_uuid(),
+    from                    uuid,
+    to                      uuid,
+    status                  text,
+    message                 text,
+    sent_timestamp          timestamptz,
+    PRIMARY KEY (id)
+);
```

Comment:
Is this meant to be here? Looks like the first migration already sets up this table.

Reply:
This file was not supposed to be here, I will be more careful.

I was trying to fix some issues with github showing old commits from way before this branch in this PR. The issue was caused by branching off from branches which were not yet merged into main. This led to some experimentation, and these files were merged in during that process.

I have deleted these files now.